src/d3/scatterplot-ctrls.ts

```diff
@@ -0,0 +1,138 @@
+    (navTile as AFRAME.Entity).setAttribute('geometry', 'primitive: plane; height: .5; width: .5');
+    if (dim === 'x'){
+      if (velocity > 0){
+        (navTile as AFRAME.Entity).setAttribute('position', tilePos.xPos);
+        (navTile as AFRAME.Entity).setAttribute('material', 'color: white; opacity: .75; src: ../assets/right_arrow.png');
+      } else {
+        (navTile as AFRAME.Entity).setAttribute('position', tilePos.xNeg);
+        (navTile as AFRAME.Entity).setAttribute('material', 'color: white; opacity: .75; src: ../assets/left_arrow.png');
+      }
+      (navTile as AFRAME.Entity).addEventListener('mousedown', () => {
+        rigPos.set(
+          rigPos.x + velocity,
+          rigPos.y,
+          rigPos.z
+        );
+      });
+    } else if (dim === 'y'){
+      if (velocity > 0){
+        (navTile as AFRAME.Entity).setAttribute('position', tilePos.yPos);
+        (navTile as AFRAME.Entity).setAttribute('material', 'color: white; opacity: .75; src: ../assets/up_arrow.png');
+      } else {
+        (navTile as AFRAME.Entity).setAttribute('position', tilePos.yNeg);
+        (navTile as AFRAME.Entity).setAttribute('material', 'color: white; opacity: .75; src: ../assets/down_arrow.png');
+      }
+      (navTile as AFRAME.Entity).addEventListener('mousedown', () => {
+        rigPos.set(
+          rigPos.x,
+          rigPos.y + velocity,
+          rigPos.z
+        );
+      });
+    } else if (dim === 'z'){
+      if (velocity > 0){
+        (navTile as AFRAME.Entity).setAttribute('position', tilePos.zPos);
+        (navTile as AFRAME.Entity).setAttribute('material', 'color: white; opacity: .75; src: ../assets/up_arrow.png');
+      } else {
+        (navTile as AFRAME.Entity).setAttribute('position', tilePos.zNeg);
+        (navTile as AFRAME.Entity).setAttribute('material', 'color: white; opacity: .75; src: ../assets/down_arrow.png');
+      }
+      (navTile as AFRAME.Entity).addEventListener('mousedown', () => {
+        rigPos.set(
+          rigPos.x,
+          rigPos.y,
+          rigPos.z + velocity
+        );
+      });
+    }
+  }
+
+export function createCtrlPanel(scatter: Scatterplot){
+    createSpeedCtrls('plus', scatter);
+    createSpeedCtrls('neg', scatter);
+    createSpeedCtrls('label', scatter);
+}
+
+function createSpeedCtrls(sign: string, scatter: Scatterplot){
+    const speedTile = document.createElement('a-entity') as AFRAME.Entity;
+    document.querySelector('[camera]').appendChild(speedTile);
+    // document.getElementById('ctrls')!.appendChild(navTile);
+    if (sign === 'plus'){
+      speedTile.setAttribute('geometry', 'primitive: plane; height: .35; width: .35');
+      speedTile.setAttribute('position', speedPos.plus);
+      speedTile.setAttribute('material', 'color: white; opacity: .75; src: ../assets/plus.png;');
+      speedTile.addEventListener('mousedown', () => {
+        scatter.DAYDREAM_NAV_SPEED = scatter.DAYDREAM_NAV_SPEED + .2;
```

Comment:
Would be good to make `.2` a global constant.

Reply:
Went ahead and made it a constant! thanks!